base/libdl.jl

```diff
@@ -205,6 +205,43 @@ if is_linux()
     end
 end # linux-only
 
+if Sys.KERNEL == :FreeBSD
```

Comment:
On more question: is sutable to use `@static` macro here? I just found that the `if is_linux` section in this file do not use `@static`. what is the reason?